### PR TITLE
OKTA-747426 - Update to 2024.06.2 RN

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Added an RN to OIE and Classic RNs for `2024.06.2` RNs
- **Is this PR related to a Monolith release?** `2024.06.2`

### Resolves:

* [OKTA-747426](https://oktainc.atlassian.net/browse/OKTA-747426)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ The limit on active event hooks per org has been increased from 10 to 25. See [E
 
 * The [oauth2/introspect](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/introspectCustomAS) endpoint hit rate limits without logging it in the System Log. (OKTA-744604)
 
+* Some users not part of the "Everyone" group were missing group memberships when querying users from the Groups API. (OKTA-700254)
```
Comment (annejuan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:70]:
```suggestion
* Some users not part of the "Everyone" group were missing group memberships when querying users from the Groups API. (OKTA-747426)
```

File: packages/@okta/vuepress-site/docs/release-notes/2024/index.md [hunk 2]
```diff
@@ -64,6 +64,8 @@ The limit on active event hooks per org has been increased from 10 to 25. See [E
 
 * The [oauth2/introspect](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/introspectCustomAS) endpoint hit rate limits without logging it in the System Log. (OKTA-744604)
 
+* Some users not part of the "Everyone" group were missing group memberships when querying users from the Groups API. (OKTA-700254)
```
Comment (annejuan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024/index.md:67]:
```suggestion
* Some users not part of the "Everyone" group were missing group memberships when querying users from the Groups API. (OKTA-747426)
```

**Review by annejuan-okta — APPROVED** [review]

**Review by annejuan-okta — APPROVED** [review]
🚢 🌶️ 🛠️

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md [hunk 3]
```diff
@@ -67,6 +67,8 @@ The limit on active event hooks per org has been increased from 10 to 25. See [E
 
 * The [oauth2/introspect](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/introspectCustomAS) endpoint hit rate limits without logging it in the System Log. (OKTA-744604)
 
+* Some users not part of the "Everyone" group were missing group memberships when querying users from the Groups API. (OKTA-747426)
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:70]:
```suggestion
* In Groups API queries, some users who weren't in the "Everyone" group were missing group memberships. (OKTA-747426)
```

File: packages/@okta/vuepress-site/docs/release-notes/2024/index.md [hunk 4]
```diff
@@ -64,6 +64,8 @@ The limit on active event hooks per org has been increased from 10 to 25. See [E
 
 * The [oauth2/introspect](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/CustomAS/#tag/CustomAS/operation/introspectCustomAS) endpoint hit rate limits without logging it in the System Log. (OKTA-744604)
 
+* Some users not part of the "Everyone" group were missing group memberships when querying users from the Groups API. (OKTA-7747426)
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024/index.md:67]:
```suggestion
* In Groups API queries, some users who weren't in the "Everyone" group were missing group memberships. (OKTA-7747426)
```

**Review by grahamsmith-okta — CHANGES_REQUESTED** [review]
Reworded the blurb slightly to clarify the "when querying" phrase, because it wasn't the users themselves who were doing the querying. Let me know if this still works.

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**